docs/star-tool.md

```diff
@@ -0,0 +1,226 @@
+`--format`    | `-f`       | Lets you configure the formatting of the output by providing one or more formatting options.
+
+Formatting options:
+
+- `i` – The name of the index to list.
+- `t` – The name of the table of the index.
+- `u` – Whether or not the index is unique.
+
+To list all indexes of the table `MyApp.Superhero`, of an existing database located at `C:\databases\mydb`, including both user indexes and system indexes and both the name of the index and whether it's unique, we run:
+
+```
+star list index -u -s -f iu C:\databases\mydb MyApp.Superhero
+```
+
+##### `info`
+
+```
+star info [options] [command]
+```
+
+The `info` command is used together with one of the following sub-commands to print info about some specific entity in a database:
+
+##### `info table`
+
+```
+star info table [options] <db> <table>
+```
+
+The `info table` sub-command prints detailed information about a table, the name of which is specified in the `<table>` argument, in the database referenced by the absolute or relative path given in the `<db>` argument. The following options are available to configure the output of the commmand:
+
+Option        | Short form | Description
+------------- | ---------- | --------------------------------------------------------------------------------------------
+`--user`      | `-u`       | Includes user columns in the output.
+`--system`    | `-s`       | Includes system columns in the output.
+`--inherited` | `-i`       | Includes inherited columns in the output.
+`--format`    | `-f`       | Lets you configure the formatting of the output by providing one or more formatting options.
+
+Formatting options:
+
+- `c` – The name of the columns of the table to list.
+- `t` – The datatype of the columns of the table to list.
+- `n` – For each column, whether it accepts null values in table cells.
+- `i` – For each column, whether it is inherited.
+- `b` – The name of the table of the column.
+
+To list all user columns of the `MyApp.Superhero` table of an existing database located at `C:\databases\mydb`, including inherited columns and listing only the names and the datatypes of columns, we run:
+
+```
+star info table -u -i -f ct C:\databases\mydb MyApp.Superhero
+```
+
+##### `info index`
+
+```
+star info index [options] <db> <index>
+```
+
+The `info index` sub-command prints detailed information about an index, the name of which is specified in the `<index>` argument, in the database referenced by the absolute or relative path given in the `<db>` argument. The following options are available to configure the output of the commmand:
+
+Option     | Short form | Description
+---------- | ---------- | --------------------------------------------------------------------------------------------
+`--format` | `-f`       | Lets you configure the formatting of the output by providing one or more formatting options.
+
+Formatting options:
+
+- `c` – The name of the column that the index is registered on.
```

Comment:
@per-samuelsson how does the tool treat multi-column indexes? And how does it treat when the columns belong to different tables?

Reply:
> @per-samuelsson how does the tool treat multi-column indexes? 

It writes out one column per line. So if you have a multi-column index Foo (Bar, Fubar), you should see:

```
info index -c Foo
Bar
Fubar
```

> And how does it treat when the columns belong to different tables?

AFAIK, we don't support that. But it will print the name of the table if you tell it so. Relates to the question I asked on Slack yesterday: https://starcounter.slack.com/archives/C5Q4Y78F5/p1576664501013700

Reply:
>AFAIK, we don't support that

Actually, we do (btw, it would be another item to think about if we decide to implement multi-column index attributes ).

Reply:
> ... btw, it would be another item to think about if we decide to implement multi-column index attributes

Please don't mix topics now. 😃